fetch.bs

```diff
@@ -3569,6 +3625,9 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    <li class=XXX><p>Needs testing: multiple `<code>Proxy-Authenticate</code>` headers, missing,
    parsing issues.
 
+   <li><p>If fetch is <a for=fetch>terminated</a>, providing a <var>reason</var>, return a
+   <a>network error</a> with <a for=response>termination reason</a> set to <var>reason</var>.
```

Comment:
I'll add this for 401 too – I just missed it.

I don't think we need enclosing steps here, it's just an extra check to avoid showing UI if the fetch has already terminated.